CONTRIBUTING.md

```diff
@@ -104,3 +105,7 @@ than waiting for a full **NCL Entry** to be completed.
 ### Adding to the NCL Index
 
 Add a new line to the `ncl/ncl_index/ncl-index-table.csv` file
+
+| NCL Function       | Description                 | Python Equivalent | Notes                          |
+|--------------------|-----------------------------|-------------------|--------------------------------|
+| `example_function <https://www.ncl.ucar.edu/>`__ | Description of the function | `python_function` | Link to `ncl_entries` notebook |
```

Comment:
Is this going to be confusing because of markdown vs csv formatting?

Reply:
Yeah I found it really clunky to edit, too, which might be problematic for outside contributors. I wonder if a yaml or json file or some other format would be easier. Even if we couldn't directly transform whatever file type into a table in the rst file, we could generate a csv in `conf.py`.

@kafitzgerald I remember you had some thoughts about this previously, do you have any opinions here?

Reply:
Yeah, I'd maybe be inclined to just point to the CSV from here and describe what goes there are a high level if needed or any special cases (these can come later to if/when we feel pain).  It's a bummer that it was kind of a pain / confusing the first time around, but I think once that's populated a bit it should be more obvious and/or we'll need to write different instructions anyway 🙃. 

It's nice to have a visual, but I don't know that it's worth spending time on getting something in here (and maintaining it) that's similar enough not to be confusing.  

Just my opinion though.

Reply:
I will add details about how to do this in the CONTRIBUTING.md (as part of #50) for now